src/renderer/main/grid-layout/grid-modules/overlays/PresetLoadOverlay.svelte

```diff
@@ -50,33 +52,30 @@
     loaded = success;
   }
 
-  let [dx, dy] = [device?.dx, device?.dy];
   let isChanged = false;
-  $: {
-    try {
-      const events = $runtime
-        .find((e) => e.dx == dx && e.dy == dy)
-        .pages[$user_input.event.pagenumber].control_elements.find(
-          (e) => e.controlElementNumber == elementNumber
-        ).events;
 
+  $: handleDeviceChange(device);
+
+  function handleDeviceChange(rt) {
```

Comment:
parameter rt is misleading and unused. Give it proper name and use it instead of using device global variable directly!

Reply:
@elsoazemelet please check this!